fuzzers/aflcc/fuzzer.py

```diff
@@ -0,0 +1,344 @@
+        utils.append_flags('CXXFLAGS', openthread_flags)
+
+    elif is_benchmark('php'):
+        php_flags = ['-D__builtin_cpu_supports\\(x\\)=0']
+        utils.append_flags('CFLAGS', php_flags)
+        utils.append_flags('CXXFLAGS', php_flags)
+
+    # For some benchmarks, we also tell the compiler 
+    # to ignore unresolved symbols. This is useful when we cannot change
+    # the build process to add a shared library for linking (which contains mocked functions: libAflccMock.so).
+    # Note that some functions are only defined post-compilation 
+    # during the LLVM passes.
+    elif is_benchmark('bloaty') or is_benchmark('openssl') or is_benchmark('systemd'):
+        unresolved_flags = ['-Wl,--warn-unresolved-symbols']
+        utils.append_flags('CFLAGS', unresolved_flags)
+        utils.append_flags('CXXFLAGS', unresolved_flags)
+
+    elif is_benchmark('curl'):
+        dl_flags = ['-ldl', '-lpsl']
+        utils.append_flags('CFLAGS', dl_flags)
+        utils.append_flags('CXXFLAGS', dl_flags)
+
+def add_post_compilation_lflags(ldflags_arr):
+    """Add additional linking flags for certain benchmarks"""
+    if is_benchmark('libjpeg'):
+        ldflags_arr += libjpeg_turbo_asm_object_files()
+    elif is_benchmark('php'):
+        ldflags_arr += ['-lresolv']
+    elif is_benchmark('curl'):
+        ldflags_arr += ['-ldl', '-lpsl', '/src/openssl/libcrypto.a', '/src/openssl/libssl.a']
+    elif is_benchmark('openssl'):
+        ldflags_arr += ['/src/openssl/libcrypto.a', '/src/openssl/libssl.a']
+    elif is_benchmark('systemd'):
+        shutil.copy(os.path.join(os.environ['OUT'], 'src/shared/libsystemd-shared-245.so'), '/usr/lib/libsystemd-shared-245.so')
+        ldflags_arr += ['-lsystemd-shared-245']
+
+def prepare_fuzz_environment(input_corpus):
+    """Prepare run for some benchmarks"""
+    afl_fuzzer.prepare_fuzz_environment(input_corpus)
+
+    # OUT env variable does not exists, it seems
+    if os.path.isfile('/out/src/shared/libsystemd-shared-245.so'):
+        shutil.copy('/out/src/shared/libsystemd-shared-245.so', '/usr/lib/libsystemd-shared-245.so')
+
+def prepare_build_environment():
+    """Set environment variables used to build benchmark."""
+    utils.set_no_sanitizer_compilation_flags()
+
+    # Update compiler flags for clang-3.8.
+    cflags = ['-fPIC']
+    cppflags = cflags + ['-I/usr/local/include/c++/v1/', 
+                         '-stdlib=libc++', '-std=c++11']
+    utils.append_flags('CFLAGS', cflags)
+    utils.append_flags('CXXFLAGS', cppflags)
+
+    # Add flags for various benchmarks
+    add_compilation_cflags()
+
+    # Setup aflcc compiler
+    os.environ['LLVM_CONFIG'] = 'llvm-config-3.8'
+    os.environ['CC'] = '/afl/aflc-gclang'
+    os.environ['CXX'] = '/afl/aflc-gclang++'
+    #  -L/ -lAflccMock -lpthread
+    os.environ['FUZZER_LIB'] = '/libAFL.a'
+
+    # Fix FUZZER_LIB for various benchmarks
+    fix_fuzzer_lib()
+
+def get_fuzz_targets():
```

Comment:
Why didn't you copy `fuzzer_utils.get_fuzz_target_binary`?

Reply:
I actually don't know what the name of the fuzz target is, because non-oss-fuzz projects don't set FUZZ_TARGET. So I just look for binaries in /src. For a few benchmarks, I did take the time to run the fuzzer and see which binary was called, so I hardcoded their names. Is FUZZ_TARGET set for all projects now?

Reply:
Oh right. Yeah I don't think so. Need to land https://github.com/google/fuzzbench/pull/263

Reply:
ok, so I'll fix this once it's landed. Thx!